### PR TITLE
Update XSD v0.20 -> v0.24 [WIP]

### PR DESCRIPTION
**Pull request by dan-palmer**

### CODE REVIEW

**Comment by dan-palmer** [comment]
All elements are present and schema validates. Need to beef up the xml feature spec before being ready to push.

File: spec/support/ETFees_schema.xsd [hunk 1]
```diff
@@ -1,308 +1,375 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dan-palmer) [inline spec/support/ETFees_schema.xsd]: Good job changing the name on this :+1:

File: app/presenters/jadu_xml/respondent_presenter.rb [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+    property :address_telephone_number, as: 'PhoneNumber', render_nil: true
     property :acas, extend: JaduXml::AcasPresenter, exec_context: :decorator
+    property :work_address, extend: JaduXml::AltAddressPresenter, class: Address
+    property :alt_phone_number, as: 'AltPhoneNumber', render_nil: true, exec_context: :decorator
 
     def acas
       represented
     end
+
+    def alt_phone_number
```
Comment (dan-palmer) [inline app/presenters/jadu_xml/respondent_presenter.rb]: :scream:

File: app/presenters/jadu_xml/alt_address_presenter.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module JaduXml
+  class AltAddressPresenter < XmlPresenter
+    include AddressProperties
```
Comment (stevegraham) [inline app/presenters/jadu_xml/alt_address_presenter.rb]: 😭

**Comment by stevegraham** [comment]
nice one

**Comment by dan-palmer** [comment]
Pushed more spec changes. Added a some nil rendering specs too.

**Pull request merged**